### ISSUE TITLE
Huobi Pro watchOrderBook error on some pairs

### ISSUE THREAD

**Issue by ndubel** [ccxt/ccxt#6748]
- Programming Language version: JS
- CCXT version: ccxtpro 0.1.40
- Exchange: Huobi Pro

I try to use `watchOrderBook`on Huobi. When I try to use it for popular pairs like ETH/BTC or BTC/USDT - everything goes fine. When I try many other pairs, for example, LINK/USDT is this case, I got this error:

```
2020-04-07T09:38:35.787Z connecting to wss://api.huobi.pro/ws
2020-04-07T09:38:36.929Z onUpgrade
2020-04-07T09:38:36.931Z onOpen
2020-04-07T09:38:36.933Z sending { sub: 'market.linkusdt.mbp.150', id: '1' }
2020-04-07T09:38:36.992Z onMessage { ping: 1586252316850 }
2020-04-07T09:38:36.993Z sending { pong: 1586252316850 }
2020-04-07T09:38:37.210Z onMessage {
  id: '1',
  status: 'ok',
  subbed: 'market.linkusdt.mbp.150',
  ts: 1586252317071
}
2020-04-07T09:38:38.934Z sending { req: 'market.linkusdt.mbp.150', id: '2' }
2020-04-07T09:38:39.225Z onMessage {
  ts: 1586252319079,
  status: 'error',
  'err-code': 'bad-request',
  'err-msg': 'invalid mbp.150.symbol linkusdt',
  id: '2'
}
/var/*/node_modules/ccxt/js/base/Exchange.js:952
            'bids': sortBy ((bidsKey in orderbook) ? this.parseBidsAsks (orderbook[bidsKey], priceKey, amountKey) : [], 0, true),
                                     ^

TypeError: Cannot use 'in' operator to search for 'bids' in undefined
    at huobipro.parseOrderBook (/var/*/node_modules/ccxt/js/base/Exchange.js:952:38)
    at huobipro.handleOrderBookSnapshot (/var/*/node_modules/ccxt.pro/js/huobipro.js:312:31)
    at huobipro.handleSubscriptionStatus (/var/*/node_modules/ccxt.pro/js/huobipro.js:476:31)
    at huobipro.handleMessage (/var/*/node_modules/ccxt.pro/js/huobipro.js:563:22)
    at WsClient.onMessage (/var/*/node_modules/ccxt.pro/js/base/Client.js:268:14)
    at WebSocket.onMessage (/var/*/node_modules/ws/lib/event-target.js:120:16)
    at WebSocket.emit (events.js:210:5)
    at Receiver.receiverOnMessage (/var/*/node_modules/ws/lib/websocket.js:801:20)
    at Receiver.emit (events.js:210:5)
    at Receiver.dataMessage (/var/*/node_modules/ws/lib/receiver.js:427:14)

```

**Comment by kroitor** [comment]
@ndubel thanks for reporting, will fix shortly!

**Comment by kroitor** [comment]
This is fixed in 0.1.42, the error handling in particular. So, it will behave as expected now. 

However, we cannot resolve the issue with a specific market on our side if Huobi does not accept real-time subscriptions for that market. 

On their website they use step0-feeds, which are not real-time (once per second). 

There is no mention about this or this type of error in Huobi API docs. It may be related to the following:

<img width="1428" alt="Screen Shot 2020-04-08 at 08 25 31" src="https://user-images.githubusercontent.com/1294454/78751270-89050500-797a-11ea-8140-1f595d10e079.png">


You may want to reach out to Huobi tech support for an explanation. We will be happy if you report back if you get a response from them.

**Comment by ndubel** [comment]
The answer is:

> https://huobiapi.github.io/docs/spot/v1/en/#market-depth this one provides all pairs;
> the other two faster update only support popular pairs
> https://huobiapi.github.io/docs/spot/v1/en/#market-by-price-incremental-update
> https://huobiapi.github.io/docs/spot/v1/en/#market-by-price-refresh-update
